plugins.json

```diff
@@ -1338,6 +1338,13 @@
         "date": "2023-02-02T23:40:28.761732-08:00",
         "sha512sum": "bb171d2b981836ddca662ada9d488008251ea13aafa4451c86c0372786fcbd7959c6d9710b5a7752fa49c94a08c4198fcc42b3c6f5b3d4fa620da474c80e691f",
         "requires": ">=22.10.6"
+      },
+      {
+        "version": "0.3.3",
+        "url": "https://github.com/quiltdata/nf-quilt/releases/download/0.3.3/nf-quilt-0.3.3.zip",
+        "date": "2023-03-08T10:56:53.523525-08:00",
+        "sha512sum": "2a4c9952090851d63010500d3777d50545f6d2521e928fc06799e2cf57fb24dbdf666e89652f74d0aad99d0ed188f62ae9573b8c014c7206da6ce4e0190c1be5",
+        "requires": "<22.11.0-edge"
```

Comment:
```suggestion
        "requires": "<22.11.0-edge"
```

Reply:
Because last time I tested,  FileSystemProvider broke due to the incompatible change to `notifyFilePublish`
```
    void notifyFilePublish(QuiltPath destination) { //, Path source=null) {
        final sess = Global.session
        /* groovylint-disable-next-line Instanceof */
        if (sess instanceof Session) {
            sess.notifyFilePublish((Path)destination) //, source)
        }
    }
```

Reply:
Here's the one-line PR I need in order to be able to support both older and newer versions:

Session.groovy
```groovy
    void notifyFilePublish(Path destination, Path source=null) {
```

https://github.com/nextflow-io/nextflow/pull/3737

Reply:
I pushed a new version of the plugin that assumes we will fix that backward incompatibility :-)

```json
        "requires": ">=22.10.6"
```

Reply:
@pditommaso Thanks for taking the compatibility fix! Can you merge the plugin now?

Reply:
The patch has been merged, but it needs to be released in the upcoming version 23.03.0-edge. Likely late next week

Reply:
That's fine, but can you merge *this* plugin now, so people can start using it with NextFlow 22 (instead of the current plugin, which has the same compatibility issues)?

Reply:
fair enough